apps/wallet-dashboard/components/dialogs/migration/MigrationDialog.tsx

```diff
@@ -31,18 +34,34 @@ export function MigrationDialog({
     isTimelocked,
 }: MigrationDialogProps): JSX.Element {
     const account = useCurrentAccount();
+    const [basicOutputs, setBasicOutputs] = useState<IotaObjectData[]>(basicOutputObjects);
+    const [nftOutputs, setNftOutputs] = useState<IotaObjectData[]>(nftOutputObjects);
+    const [isPartialMigration, setIsPartialMigration] = useState<boolean>(false);
     const [txDigest, setTxDigest] = useState<string>('');
     const [view, setView] = useState<MigrationDialogView>(MigrationDialogView.Confirmation);
 
     const {
         data: migrateData,
         isPending: isMigrationPending,
         isError: isMigrationError,
-    } = useMigrationTransaction(account?.address || '', basicOutputObjects, nftOutputObjects);
-
+        error,
+    } = useMigrationTransaction(account?.address || '', basicOutputs, nftOutputs);
+    console.log(isMigrationError, error);
     const { mutateAsync: signAndExecuteTransaction, isPending: isSendingTransaction } =
         useSignAndExecuteTransaction();
 
+    useEffect(() => {
+        if (isMigrationError) {
```

Comment:
can we be sure this is the only possible error we could get? isnt there a type of error we could use to filter out only max size errors?

Reply:
I might not be fully seeing the flow of the code, but wouldnt `isMigrationError` become false as soon as the tx is correct?